openslides_backend/permissions/permission_helper.py

```diff
@@ -17,7 +17,6 @@ def has_perm(
             FullQualifiedId(Collection("user"), user_id),
             [
                 f"group_${meeting_id}_ids",
-                "guest_meeting_ids",
```

Comment:
Is this function adapted to the new user modelling?

Reply:
I think we have to remove the "guest_meeting_ids"

Reply:
Yes, I adapted it in my initial PR because all kinds of tests failed without it.